python/lsst/pipe/supertask/super_task.py

```diff
@@ -223,6 +223,111 @@ def write_schemas(self, butler, clobber=False, do_backup=True):
                 self.log.info("Writing schema %s" % schema_dataset)
                 butler.put(catalog, schema_dataset)
 
+    def get_resource_config(self):
+        """Return resource configuration for this task.
+
+        Returns object of type `resource_config.ResourceConfig`or None if resource configuration
+        is not defined for this task.
+        """
+        return getattr(self.config, "resources", None)
```

Comment:
Either that or you can add "resources" field to any other config class if you don't want to inherit from ConfigWithResource. I'm open to any suggestion how to make it more usable.